src/aleph/vm/orchestrator/views/operator.py

```diff
@@ -143,9 +143,8 @@ async def stream_logs(request: web.Request):
     queue: asyncio.Queue = asyncio.Queue()
     try:
         ws = web.WebSocketResponse()
+        await ws.prepare(request)
         try:
-            await ws.prepare(request)
-
```

Comment:
Why move this part ?

Reply:
Because if the prepare() raise an exception, e.g when it cannot negociate a websocket connection, when `close()`  is called it raise following error:
`RuntimeError: Call .prepare() first`